source/clear-linux/virtual-environments/kvm.rst

```diff
@@ -1,57 +1,156 @@
 .. _kvm:
 
-Using KVM
-#########
+Running Clear Linux as a KVM guest OS
+#####################################
 
-The easiest way to get started in a virtualized environment is to download
-a recent KVM image from the `images`_ directory. Here you'll find a kvm
-image file, the UEFI firmware helper, and the KVM start helper script.
+To run |CL| in a virtualized environment using abbr:`KVM (Kernel-based
```

Comment:
Change to |CLOSIA| to have the full trademarked name at least once on the document.